.github/workflows/helper/configureDevClient.js

```diff
@@ -0,0 +1,39 @@
+fs = require('fs');
+
+function patchFile(path, find, replace) {
+  let data = fs.readFileSync(path, 'utf8');
+  data = data.replace(find, replace);
+  fs.writeFileSync(path, data);
+}
+
+const command = process.argv[2];
+
+if (command === 'setBundleIdentifier') {
+  patchFile(
+    'app/app.json',
+    '"ios": {',
+    '"ios": {"bundleIdentifier":"com.swmansion.app",'
+  );
+
+  patchFile(
+    'app/app.json',
+    '"android": {',
+    '"android": {"package": "com.swmansion.app",'
+  );
+}
+
+if (command === 'setupFabricIOS') {
+  patchFile(
+    'app/ios/Podfile.properties.json',
+    '{',
```

Comment:
fishy but ok 🐟

Reply:
Fixed in: https://github.com/software-mansion/react-native-reanimated/pull/4818/commits/6d31fecace21848784643ebc9c69f2d74c133f10